### PR TITLE
feat: support date and datetime types.

### PR DESCRIPTION
**Pull request by ya7010**
Added logic for using date and datetime types in pydantic models.

This allows support for date-related input.

### CODE REVIEW

**Comment by elda27** [comment]
I understand pydantic has an ability to convert string to datetime.
Could you provide a detail in new issue.

**Comment by yassun7010** [comment]
Add Issues. https://github.com/elda27/pydantic_argify/pull/7

**Comment by yassun7010** [comment]
@elda27 

Fixed the code to treat it as str type.

File: pydantic_argify/parse.py [hunk 1]
```diff
@@ -352,11 +351,8 @@ def _parse_shape_args(name: str, field: FieldInfo) -> dict:
     kwargs: dict[str, Any] = {}
     kwargs["type"] = field.annotation
 
-    if kwargs["type"] is datetime.datetime:
-        kwargs["type"] = SchemaValidator({"type": "datetime"}).validate_python
-
-    elif kwargs["type"] is datetime.date:
-        kwargs["type"] = SchemaValidator({"type": "date"}).validate_python
+    if kwargs["type"] in (datetime.datetime, datetime.time, datetime.date):
```
Comment (elda27) [inline pydantic_argify/parse.py]: Do you forget datetime.timedelta?

**Comment by elda27** [comment]
Thanks!

**Pull request merged**